NAMESPACE

```diff
@@ -21,27 +21,28 @@ export(export_data)
 export(get_articles)
 export(get_links)
 export(get_memberships)
+export(get_pdf)
```

Comment:
Is `get_` here operating at the same level as other `get_` functions?

Reply:
Some are some not, you are correct. I have moved the ones on the same level to a new script and renamed get_pdf() to convert_pdf()  for clarity.